README.md

```diff
@@ -201,6 +201,26 @@ ninja gtests_libdaemon
 ```
 And the unit-test executables are at `$BUILD/dbms/gtests_dbms`, `$BUILD/libs/libcommon/src/tests/gtests_libcommon` and `$BUILD/libs/libdaemon/src/tests/gtests_libdaemon`.
 
+## Run Sanitizer Tests
+
+TiFlash supports testing with thread sanitizer and address sanitizer.
+
+To generate unit test executables with sanitizer enabled:
+
+```shell
+cd $BUILD
+cmake $WORKSPACE/tiflash -GNinja -DENABLE_TESTS=ON -DCMAKE_BUILD_TYPE=ASan # or TSan
+ninja gtests_dbms
+ninja gtests_libcommon
+ninja gtests_libdaemon
+```
+
+There are known false positives reported from leak sanitizer (which is included in address sanitizer). To suppress these errors, set the following environment variables before running the executables:
+
+```
+LSAN_OPTIONS=suppressions=/path/to/tiflash/test/sanitize/asan.suppression
```

Comment:
do we have any pregenerated asan.suppression file?

Reply:
yes, following the path.

Reply:
maybe use `LSAN_OPTIONS=suppressions=$WORKSPACE/tiflash/test/sanitize/asan.suppression`

Reply:
@JaySon-Huang that's clear enough to me.